src/Evolution/Systems/Cce/OptionTags.hpp

```diff
@@ -319,11 +328,34 @@ struct EndTime : db::SimpleTag {
   }
 };
 
+/// \brief Represents the final time of a CCE evolution that should just proceed
+/// until it receives no more boundary data and becomes quiescent.
+struct NoEndTime : Tags::EndTime, db::SimpleTag {
+  using type = double;
+  using option_tags = tmpl::list<>;
+
+  static constexpr bool pass_metavariables = false;
+  static double create_from_options() {
+    return std::numeric_limits<double>::infinity();
```

Comment:
In current code, this won't FPE because it is only used in the comparison operator to see if the current time is still less than the end time. It would if we tried to do math with it in the future.